operators/src/util/async_util.rs

```diff
@@ -0,0 +1,40 @@
+use rayon::ThreadPool;
+use std::sync::Arc;
+use tokio::task::JoinHandle;
+
+/// A wrapper around `tokio::task::spawn_blocking` that wraps the
```

Comment:
shouldn't this be in datatypes (aka _base_) s.t. it can also be used there? Johannes wanted to move processing in there. ThreadPool must move there too then

Reply:
Datatypes doesn't know about `tokio`, so we only need it in operators and services